packages/trees/backend/sirius-components-collaborative-trees/src/main/java/org/eclipse/sirius/components/collaborative/trees/api/IDropTreeItemHandler.java

```diff
@@ -24,7 +24,7 @@
  */
 public interface IDropTreeItemHandler {
 
-    boolean canHandle(Tree tree);
+    boolean canHandle(IEditingContext editingContext, Tree tree);
```

Comment:
Added the `editingContext` parameter for handlers that need to access it (e.g. to retrieve the view description of the explorer tree)